README.md

```diff
@@ -241,6 +242,27 @@ These settings control the movement of files to and from object storage.
 ### File System settings
 - **Storage File System Selection**: The backend filesystem to be used. This is also used for the background transfer tasks when the main alternative_file_system_class variable is not set.
 
+### Pre-Signed URLs Settings
+- **Enable Pre-Signed URLs**: Enable redirect requesting content from external storage.
+- **Pre-Signed URL expiration time**: The time after which the <b>Pre-Signed URL</b> should expire.
+- **Minimum size for Pre-Signed URL (KB)**: Minimum file size required to redirect requests to an external storage.
+- **Pre-Signed URL whitelist**: Specify file extensions eligible to generate a <b>Pre-Signed URL</b>
+- **Signing method**: Define the desired client to generate <b>Pre-Signed URLs</b>.
+    * Options available:
+        - **S3**
+        - **CloudFront**
+    * S3: Inherits the settings from [Amazon S3 settings](#amazon-s3-settings)
+    * Cloudfront: It requires to create a [Cloudfront Distribution](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/distribution-web-creating-console.html)
+        * **DOMAIN (inc. https://)**: Domain name where the content requests will be redirected to.
+        * **Key_Pair ID from AWS**: Key to identify your trusted signers. [Creating CloudFront Key Pairs](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/private-content-trusted-signers.html#private-content-creating-cloudfront-key-pairs)
+        * **PRIVATE Key .pem**:
+            * can be one of the following:
+                * A file name with the pem extension e.g.: cloudfront.pem
```

Comment:
I think it would be good to add a path where the file should be located. 
The same as it's described on objectfs settings page.

Reply:
updated